youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -4,8 +4,8 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: youtube-dlg 0.4\n"
-"POT-Creation-Date: 2017-06-15 17:13+EEST\n"
-"PO-Revision-Date: 2017-06-15 17:14+EEST\n"
+"POT-Creation-Date: 2017-09-26 00:44+JST\n"
```

Comment:
You should change only the **PO-Revision-Date** and NOT the **POT-Creation-Date** (that's the date the original translation file was created). Please set the **POT-Creation-Date** back to the old value *2017-06-15 17:13+EEST*.

Everything else is fine.